changelog.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- version: 1.5.0
+  date: '2020-12-14'
+  added:
+  - Add `#or_nil` to support nullifying invalid values (@ianks)
```

Comment:
@ianks please remove this part because we update changelogs via PR merge commit messages, and yes, this should be documented in that file and contribution guidelines, which I'll do shortly 🙂